pkg/virt-controller/watch/drain/disruptionbudget/disruptionbudget.go

```diff
@@ -289,10 +288,6 @@ func (c *DisruptionBudgetController) Execute() bool {
 
 func (c *DisruptionBudgetController) execute(key string) error {
 
-	if !c.podDisruptionBudgetExpectations.SatisfiedExpectations(key) {
```

Comment:
do we not want to keep this check to see if the expectation is satisfied?

Reply:
Definitely I just wanted to move it further down. :+1:

Reply:
Back where it should be.